catalog/core/catalog-core-standardframework/src/test/groovy/ddf/catalog/impl/operations/SourceOperationsTest.groovy

```diff
@@ -399,6 +400,11 @@ class SourceOperationsTest extends Specification {
         !available
     }
 
+    /**
+     * Please see <a href="https://codice.atlassian.net/browse/DDF-2440">DDF-2440</a> for more details
```

Comment:
Could you add a `TODO: ` to both of these comments so they're easier to track in the codebase? I think that's still our preferred style.